src/pages/MangaBuddy/main.ts

```diff
@@ -0,0 +1,85 @@
+import { pageInterface } from '../pageInterface';
+
+export const MangaBuddy: pageInterface = {
+  name: 'MangaBuddy',
+  domain: 'https://mangabuddy.com',
+  languages: ['English'],
+  type: 'manga',
+  isSyncPage(url) {
+    return j.$('div.reading-box').length > 0;
+  },
+  isOverviewPage(url) {
+    return j.$('div.read-box').length > 0;
+  },
+  getImage() {
+    return j.$('div.img-cover img').attr('src');
+  },
+  sync: {
+    getTitle(url) {
+      return j.$('div.breadcrumbs-item:nth-child(2) span').text();
+    },
+    getIdentifier(url) {
+      return utils.urlPart(url, 3);
+    },
+    getOverviewUrl(url) {
+      return utils.absoluteLink(MangaBuddy.sync.getIdentifier(url), MangaBuddy.domain);
+    },
+    getEpisode(url) {
+      const episodePart = utils.urlPart(url, 4);
+
+      const temp = episodePart.match(/chapter-(\d+)/im);
+
+      if (!temp) return NaN;
+
+      return Number(temp[1]);
+    },
+    nextEpUrl(url) {
+      const next = j.$('#btn-next').attr('href');
+
+      if (next === '#') return undefined;
+
+      return utils.absoluteLink(next, MangaBuddy.domain);
+    },
+  },
+  overview: {
+    getTitle(url) {
+      return j.$('div.detail h1').text();
+    },
+    getIdentifier(url) {
+      return utils.urlPart(url, 3);
+    },
+    uiSelector(selector) {
+      j.$('div.summary')
+        .first()
+        .after(
+          j.html(
+            `<div id= "MALSyncheading" class="heading"> <span class="text-highlight">MAL-Sync</span></div><div id="malthing">${selector}</div>`,
+          ),
+        );
+    },
+    list: {
```

Comment:
The list is by default only shows part of the chapters on the overview page. The list extraction can be retriggered with `page.handleList();` function. Easierst would be to just listen on the show more button for a click and then get the new list  

https://github.com/MALSync/MALSync/blob/629efad20289c06df61ecf7f6efef367aba67ac0/src/pages/AnimeOnsen/main.ts#L129-L133

Reply:
Yeah, that fixed it. The series I initially tested had fewer chapters so I guess it didn't have any problems. How do you usually test pages before adding them?